### DISCUSSION TITLE
Disable not installed notification?

### DISCUSSION THREAD

**Discussion by dailytabs** [microsoft/vscode-docker#3907] [Q&A]
With Docker Desktop in WSL 2, the docker binaries don't exist until Docker Desktop is running. If it is not running when I open VS Code, vscode-docker complains that docker is not installed with a notification.

I wish to disable this notification, as it is not helpful. Searching the settings for "install", "warning", and "notification" found nothing.

Is it possible?

**Comment by bwateratmsft** [comment]
Is this with Docker Desktop installed on Windows or within the WSL "virtual machine"?
